x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/user_name.tsx

```diff
@@ -47,6 +50,13 @@ const UserNameComponent: React.FC<Props> = ({
   const userName = `${value}`;
   const isInTimelineContext = userName && eventContext?.timelineID;
   const { openRightPanel } = useExpandableFlyoutApi();
+  const {
+    services: { application },
+  } = useKibana();
+
+  const currentAppId = useObservable(application.currentAppId$);
+
+  const isAppSecurity = useMemo(() => currentAppId === APP_UI_ID, [currentAppId]);
```

Comment:
We are doing the same thing in different places, could you create a reusable hook (`useIsSecurityApp` or something)?

Reply:
done here : https://github.com/elastic/kibana/pull/199279/commits/5824506b8930d6c0e1ae468c7fb1ec77c413a83e